### ISSUE TITLE
Better overview of topics

### ISSUE THREAD

**Issue by eddelbuettel** [ess-intro/ess-intro#4]
The draft list of topics was carried over from our initial collabedit page to [this wiki page](https://github.com/ess-intro/ess-intro/wiki/Topics). 

A good start, but it needs an update for the next step(s):
- we need 'who is working on what' for the ongoing projects
- we probably want a nicer/simpler/... discussion space for 'who wants to work on what'
- should we stick with the wiki (as that is easier than pull-requesting)
- or should we go wild-west and set-up a custom repo where we all have write access and just edit directly?

My preference is wiki.  Other ideas?  Anybody want to start it once it takes shape?

**Comment by greg-minshall** [comment]
Dirk, i was thinking of brainstorming on org-mode in ess-intro/presentation-org-mode.  but, i'd be happy to do it somewhere else.

btw, i started presentation-org-mode, but there were a couple of other people who had volunteered, and i hope they do a lot of contributing!  i just wanted to get things going.  cheers.

**Comment by eddelbuettel** [comment]
Thumbs up!  I think within a given topic (_i.e._ for your work on org-mode presentations) nothings beats a repo. It's clear, workflows are understood, we have issue tickets.

It was more a 'meta topic' about all our 'topics' and the who-does-what-when-where and who-may-review-a-topic and all that which may be more 'scratchpad' alike and I am musing where to put it ...  Best guess so far is wiki page which is actually not bad.

**Comment by aezarebski** [comment]
To throw another spanner in the works, I'll suggest that we ditch the wiki and just run a bunch of threads in different issues. Well, keep the wiki for more stable stuff, but move brainstorming/discussion into different issues with a tag indicating they are intended to be used as such.

Of course, the problem with this is that it is not the same issue tracker as the individual respos but for less focused stuff I think it would work.

**Comment by eddelbuettel** [comment]
Yes I am of course fully ok with issues for discussions --  but I have also experienced "diluted" repos where it was harder to follow stuff in an issue.  Issues are good for clearly identified "discussion threads" around a single (or focussed) topic.  We also need a top-level "display" or "map" for the topics and the wiki format ("edit in place") may be better.  My $0.02.

**Comment by plantarum** [comment]
Hi,

I like using issues to deal with discussions related to specific 
presentations. For larger/broader discussions, I think issue's 
filed against Dirk's ESS-INTRO repo would be a good place.

I find wikis are great for documenting the results of a 
discussion, but get confusing when used for the discussion itself. 
The converse is also true: issues are great for discussions, but 
when I need to know what the outcome/decision of a discussion was, 
it can be really frustrating to have to walk through a long thread 
to find it.

Best,

Tyler

Dirk Eddelbuettel writes:

> Yes I am of course fully ok with issues for discussions --  but 
> I have also experienced "diluted" repos where it was harder to 
> follow stuff in an issue.  Issues are good for clearly 
> identified "discussion threads" around a single (or focussed) 
> topic.  We also need a top-level "display" or "map" for the 
> topics and the wiki format ("edit in place") may be better.  My 
> $0.02.


-- 
Tyler Smith
plantarum.ca

**Comment by greg-minshall** [comment]
i like issues.  but i've not had the experience of trying to follow a
bunch in one repo.  so, i hope the "user experience" scales well in this
case.  i do like being able to hit "Close" on a particular thing, and,
hopefully, in an entropy-destroying way, see the number of active issues
decline over time.

**Comment by eddelbuettel** [comment]
See the [Topics wiki page](https://github.com/ess-intro/ess-intro/wiki/Topics) for an attempt at a bit more structure.

**Comment by greg-minshall** [comment]
Dirk, thanks.  is there a github way of signing up to see all activity (commits, etc.) in any sub-project?  doing that might provide me motivation to get my stuff better organized.  (sort of, "self-shaming as a motivational force". :)

**Comment by greg-minshall** [comment]
another thought for the assembled masses: should we each name our principle file `README.foo`?  then people coming to the "ess and org" mode folder would, i'm pretty sure, be confronted with a formatted version, rather than having to decipher the .org markup.  (and, hopefully for *me*, the same for markdown.)

alternatively, i assume github has something like gitlab, where one can set up a checkin to produce (via CI/CD) an output page on `ESS-IS-GREAT.github.io`, or some such place?  that might be more work to set up, but would possibly give the casual viewer a better sense of what is there?

**Comment by eddelbuettel** [comment]
> is there a github way of signing up to see all activity (commits, etc.) in any sub-project?

I think subscribing to it does that that, i.e. the 'watch' button/menu.

> README.foo?

I only use .md which I produce statically. I include a static pdf as well.

> can one produce ...

Yes a lot of people are completely gaga over GitHub Actions. I consider that a distraction.  We should first and foremost keep it _simple_ and not put up additional barriers to participation by raisng expectations that _yet another_ process needs to be learned and mastered.

**Comment by greg-minshall** [comment]
> I think subscribing to it does that that, i.e. the 'watch' button/menu.

Dirk, thanks.  unless i'm missing a watch, i guess on a per-repo basis, not on the entire collection of repos.

yes, keeping simple is good.  and, i guess now, "let a thousand flowers" (or, presentation methods) bloom.

**Comment by eddelbuettel** [comment]
GitHub can be mysterious that way. I think I am auto-subscribed as I started the org.  And then _I_ of course forget not everybody is.

I have been thinking we need a better 'roll call' "soon". We have a few almost ready, and a few (coug, cough) in "need to be shamed in action" mode.  But we also know shaming never works so we'd never do that, right?  :sunglasses:

**Comment by lebensterben** [comment]
I just come across this very old answer by @eddelbuettel on StackOverflow: https://stackoverflow.com/a/6848914/5715571

Each point mentioned in the answer can be expanded to a tutorial on a specific functionality.

This could serve well as an outline.

**Comment by eddelbuettel** [comment]
:heart: 

I had _of course_ long forgotten about that, being a decade old.  Still true though :)
